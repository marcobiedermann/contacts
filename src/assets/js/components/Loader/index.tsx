import React, { FC } from 'react';

const Loader: FC = () => <div>loading â€¦</div>;

export default Loader;
