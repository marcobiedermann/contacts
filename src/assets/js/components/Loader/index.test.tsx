import { render } from '@testing-library/react';
import React from 'react';
import Loader from '.';

describe('loader component', () => {
  it('should render correctly', () => {
    expect.assertions(1);

    const { asFragment } = render(<Loader />);

    expect(asFragment()).toMatchSnapshot();
  });
});
