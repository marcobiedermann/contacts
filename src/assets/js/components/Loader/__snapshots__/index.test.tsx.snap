// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`loader component should render correctly 1`] = `
<DocumentFragment>
  <div>
    loading â€¦
  </div>
</DocumentFragment>
`;
